docs/concepts/what-is-an-event-history.md

```diff
@@ -15,6 +15,6 @@ An append-log of [Events](/concepts/what-is-an-event) for your application.
 
 **Event History limits**
 The Temporal Cluster stores the complete Event History for the entire lifecycle of a Workflow Execution.
-There is a hard limit of 50,000 Events in a Workflow Execution Event History, as well as a hard limit of 50 MB in terms of size.
+[There is a hard limit of 50,000 Events](/concepts/what-is-a-workflow-execution/#limits) in a Workflow Execution Event History, as well as a hard limit of 50 MB in terms of size.
```

Comment:
I'm worried about us stating the exact limit in so many places. Because we may update it soon, I think we should instead say that the limit exists and link to the main limits page.